src/components/Article.jsx

```diff
@@ -1 +1,36 @@
 // export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const today = date;
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <>
+      <div className="box">
+        <h2 className="title" data-cy="title">
+          {title}
+        </h2>
+
+        <p className="subtitle">
+          <time dateTime={dateISO} data-cy="date">
+            {dateLocaleString}
+          </time>
+        </p>
+
+        <p data-cy="text">
+          {text}
+        </p>
+      </div>
```

Comment:
your component must have some container after it is unwrapped. React.fragment disappears after the component is unwrapped.